src/grammar.coffee

```diff
@@ -33,6 +38,8 @@ placeholder = p:[a-zA-Z0-9]+ {
   return p.join("");
 }
 
+default = emptyString / string
```

Comment:
I wonder if there would be a use case to provide a default that is actually a file?